app/services/idv/flows/recovery_flow.rb

```diff
@@ -1,47 +0,0 @@
-module Idv
-  module Flows
-    class RecoveryFlow < Flow::BaseFlow
-      STEPS = {
-        recover: Idv::Steps::RecoverStep,
-        overview: Idv::Steps::OverviewStep,
```

Comment:
This view can be removed as well:

https://github.com/18F/identity-idp/blob/stevegsa-rm-ial2-recovery/app/views/idv/doc_auth/overview.html.erb

I'd expect in the end of all of these string files should be removable?

https://github.com/18F/identity-idp/tree/stevegsa-rm-ial2-recovery/config/locales/recover